### DISCUSSION TITLE
new method ImportColumn::make('temp_field')->ignoreForImport()

### DISCUSSION THREAD

**Discussion by juanborras** [filamentphp/filament#12749] [Ideas]
I'm trying to import a student csv file where a required field (unit) for the import process is required,  but it is not required to be imported into the db (in fact it is not mapped). The import process fails because the $this->record ends up with the 'unit' key... any ideas?

**Comment by Joemires** [comment]
You can use the fillRecordUsing method to achieve that
```php
ImportColumn::make('some_column_you_dont_want_to_import')
    ->fillRecordUsing(fn (Model $record, string $state) => null),
```

**Reply by juanborras** [reply]
Thanks!
